nginx/main.conf.liquid

```diff
@@ -10,14 +10,21 @@ events {
 
 http {
   lua_shared_dict whitelist 10m;
+  lua_shared_dict prometheus_metrics 10M;
 
   lua_code_cache {{ lua_code_cache }};
 
   lua_package_path "{{ lua_path }};;";
   lua_package_cpath "{{ lua_cpath }};;";
 
   init_by_lua_block {
+    local prometheus_metrics = ngx.shared.prometheus_metrics
```

Comment:
This is not used.

Reply:
I thought it was more efficient declaring local variables in Lua... Well looks like I was wrong :)